src/player.py

```diff
@@ -1,2 +1,14 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+class Player:
+    def __init__(self, room):
+        self.room = room
+
+    def change_room(self, new_room):
```

Comment:
for more complicated situations this is a good idea (for instance maybe changing room would change more than one attribute of player. for a simple situation you can simply access your player instance's room as: `player_instance.room = new_room`

Reply:
Noted! 👍